packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/simulation/Simulation.java

```diff
@@ -76,6 +83,12 @@ public class Simulation extends TerariumAsset {
 	@TSOptional
 	private UUID projectId; // TODO this can probably be joined to the project table soon?
 
+	@OneToMany(fetch = FetchType.EAGER, cascade = CascadeType.ALL)
+	@OrderBy("createdOn DESC")
+	@ToString.Exclude
+	@JsonManagedReference
+	private List<SimulationUpdate> updates = new ArrayList<>();
```

Comment:
Do these need to be handled in the `Clone` operation? right now these will be lost.

Reply:
Fixed